src/app/api/services/guards/menu-guard.service.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { TestBed, inject } from '@angular/core/testing';
+
+import { MenuGuardService } from './menu-guard.service';
+
+describe('ValidMenuGuardService', () => {
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [MenuGuardService]
+        });
+    });
+    it('should be created', inject([MenuGuardService], (service: MenuGuardService) => {
+        expect(service).toBeTruthy();
+    }));
+});
```

Comment:
just one test?

Reply:
Yup... definitely need more testing and in the describe it should go the name of the class.